File: RxHttp/src/main/java/com/github/kongpf8848/rxhttp/request/PutRequest.java
package com.github.kongpf8848.rxhttp.request;

import android.app.Activity;
import android.content.Context;

import androidx.fragment.app.Fragment;


public class PutRequest extends PostRequest {

    public PutRequest(Context context) {
        super(context);
    }

    public PutRequest(Activity activity) {
        super(activity);
    }

    public PutRequest(Fragment fragment) {
        super(fragment);
    }

}

File: RxHttp/src/main/java/com/github/kongpf8848/rxhttp/request/PutRequest.kt
package com.github.kongpf8848.rxhttp.request

import android.app.Activity
import android.content.Context
import androidx.fragment.app.Fragment

class PutRequest<T> : PostRequest<T> {
    constructor(context: Context) : super(context)
    constructor(activity: Activity) : super(activity)
    constructor(fragment: Fragment) : super(fragment)
}